cicecore/shared/ice_init_column.F90

```diff
@@ -2948,6 +2951,10 @@ subroutine init_zbgc
       endif
       if (.NOT. dEdd_algae) nbtrcr_sw = 1
 
+      ! tcraig, added 6/1/21, why is nbtrcr_sw set here?
```

Comment:
I took a quick look at the code and it wasn't obvious.  I suspect that there's 2 considerations which conflict with each other, making it confusing:
1.  The need the count up all of the tracer indices, to make sure we have the right number 
2.  An attempt to keep the BGC code as modular as possible

Reply:
This fixes an issue in the current bgc sw implementation, but there are still others are pointed out in #437.